### PR TITLE
New Mac/iOS/Android website content (RFC)

### PR DESCRIPTION
**Pull request by micahgodbolt**
This is the mega-branch for pulling in dozens of content changes targeting mac/ios/android/cross-plat for build.

#### Add control markdown files

	1. goto apps\fabric-website\src\pages\Controls\TextPage\docs
	2. create 'mac' and 'windows' folder.
	3. create TextImplementation.md, TextOverview.md and TextUsage.md in each folder

	
#### Add default sections to page

	1. Goto apps\fabric-website\src\pages\Controls\TextPage\TextPage.doc.ts
	2. copy ios/android key: object and change to 'mac' and 'windows'
	3. update URL in overview
	4. to add 'usage' key with require('!raw-loader!<path to file>')

#### Add custom sections (implementation or other custom sections)

	1. goto apps\fabric-website\src\pages\Controls\TextPage\TextPage.tsx
	2. Update _otherSections with switch cases
		a. case 'mac'
		b. case 'windows'
		c. include sectionName, editURL and content
	3. update editURL and content path to 'mac' and 'windows'

#### Add link to left nav

	1. goto apps\fabric-website\src\SiteDefinition\SiteDefinition.pages\Controls\mac.tsx | windows.tsx
	2. Determine if category is already added (Basic Inputs, Pickers, Items and Lists etc)
		a. Add category if necessary
	3. Add item to pages array. Set title, url and getComponent callback function

#### Update 'related controls'

    1. goto apps\fabric-website\src\pages\Controls\TextPage\docs\TextRelated.md
    2. Add links to mac/windows controls. Add any other related controls. 

## New Markdown Widget

```markdown
<DisplayToggle onText="Dark" offText="Light" label="Theme Switcher">

## Stuff 1

<img className="off" src="https://via.placeholder.com/150/000000/ccc?text=LightMode" />
<img className="on" src="https://via.placeholder.com/150/ccc/000000?text=DarkMode" />

## Stuff 2

<img className="off" src="https://via.placeholder.com/300/000000/ccc?text=LightMode" />
<img className="on" src="https://via.placeholder.com/300/ccc/000000?text=DarkMode" />

</DisplayToggle>
```

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12482)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
I've opened this up for review now as we want to ship these updates soon.

**Comment by jennbost** [comment]
@micahgodbolt, could you update this homepage copy. In the Powering Microsoft 365 section, replace "globalization" with "internationalization". I've worked with the CELA team to standardize the terminology we use for this and this was the last place I missed updating the reference. Thanks!

As in, "Build your own apps using the same open source components we do—with accessibility, internationalization, and performance included."

File: apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/windows/GetStartedOverview.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+WinUI is a user interface layer that contains native controls and styles for building Windows apps. It embodies Fluent Design, giving each Windows app the polished feel that customers expect.
+
+To build Fluent experiences on Windows using native frameworks, see [WinUI documentation](https://microsoft.github.io/microsoft-ui-xaml/).
+
+### Fluent UI React Native
+
+Fluent UI React Native is an [open-source](https://github.com/microsoft/fluentui-react-native) library built with JavaScript and [React Native components](https://microsoft.github.io/react-native-windows/). Use it to create Fluent experiences that work across Windows and across other platforms. It contains information about colors and typography, as well as custom controls and customizations.
+
+To build Fluent experiences on Windows using cross-platform frameworks, see [Cross-platform Get started page](#/get-started/crossplatform).
```
Comment (ejlayne) [inline apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/windows/GetStartedOverview.md:13]: @micahgodbolt - Jennifer has requested that this link should read "..., see [Get started with cross-platform](...)." See this bug if you have any questions. https://dev.azure.com/microsoftdesign/fluentui-native/_workitems/edit/7367

Comment (micahgodbolt) [reply apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/windows/GetStartedOverview.md:13]:
```suggestion
To build Fluent experiences on Windows using cross-platform frameworks, see [Get started with cross-platform](#/get-started/crossplatform).
```

**Comment by micahgodbolt** [comment]
HERE GOES!

File: packages/example-app-base/index.html [hunk 2]
```diff
@@ -46,9 +46,9 @@
       }
 
       if (useMinified) {
-        scripts.push('demo-app.min.js');
+        scripts.push('demo.min.js');
```
Comment (ecraig12345) [inline packages/example-app-base/index.html:49]: Was there a reason for this change? It broke aka.ms/fabricdemo and individual packages' PR deploy.

**Comment by micahgodbolt** [comment]
It was broken when trying to run it locally. try npm start for example-app-base


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Pull request merged**